library/cwm/src/lib/cwm/popup.rb

```diff
@@ -34,6 +36,12 @@ def should_open_dialog?
       true
     end
 
+    # Popup does not allow nil, so overwrite Dialog default value.
+    # @return [String,nil] The dialog title.
```

Comment:
If it does not allow `nil`, then the return should be modified accordingly, isn't it?

Reply:
yes, you are right. Nil cause libyui red warning.